### DISCUSSION TITLE
Keep Macbook monitor always turned off

### DISCUSSION THREAD

**Discussion by alb-g** [MonitorControl/MonitorControl#1479] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

It seems impossible to keep the MacBook display turned off when an external monitor is connected.

As many other users, **I only use the external monitor but I don't close the lid** of the Macbook (I use the keyboard).
I set screen mirroring and I dim the Macbook monitor with the function keys/touchbar until it shuts down.
I use MonitorControl just to control the external monitor.

**After every sleep and power on, the MacBook monitor returns to 8%** (just above the power off level).
**So I have to manually turn it off again.**

I tried all possible combinations of settings,
including "Apply last saved values to the display" and "Assume last saved settings are valid".

### Describe the solution you'd like

An option that allows to keep the MacBook display turned off when an external monitor is connected, would be useful.

### Describe alternatives you've considered

I have no idea  :-(

**Comment by waydabber** [comment]
Hi there - you can use [BetterDisplay](https://betterdisplay.pro) to turn off your internal display completely. I don't plan to add that functionality to MonitorControl.
